apps/web/src/components/SkillBrowser/SkillBrowserDialog.tsx

```diff
@@ -153,7 +153,13 @@ const SkillBrowserDialog = ({
         <Dialog.Header subtitle={subtitle}>{title}</Dialog.Header>
         <Dialog.Body>
           <FormProvider {...methods}>
-            <form onSubmit={methods.handleSubmit(handleAddSkill)}>
+            <form
+              onSubmit={async (e) => {
```

Comment:
Sorry, I'm still able to replicate this.  I have to press enter multiple times, but I can still trigger it.

https://github.com/user-attachments/assets/3ab1998e-a199-4f1b-a0b2-78bdc6490641

Reply:
Oops, I had the `stopPropagation` call in the wrong spot :sweat_smile: 

[move stop propagation to before mutation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13452/commits/517002cbe9dcd03b65f641e3c555079202c31bfb)